CMakeLists.txt

```diff
@@ -59,10 +59,10 @@ set(CORENRN_ClangFormat_FILES_RE
     "^.*\\\\.cu$$" "^.*\\\\.[chi]pp$$"
     CACHE STRING "List of regular expressions matching C/C++ filenames" FORCE)
 set(CORENRN_ClangFormat_EXCLUDES_RE
-    "${CORENRN_PROJECT_SOURCE_DIR}/external/.*$$"
+    "external/.*$$"
```

Comment:
```suggestion
    ""
```

Reply:
that's right, paths are relative to ${CORENRN_PROJECT_SOURCE_DIR} now